src/common/module-type.ts

```diff
@@ -6,7 +6,8 @@ import { $enum } from "ts-enum-util";
  */
 export enum ModuleType {
   CURRENCY = 0,
-  COLLECTION = 1,
+  COLLECTON = 1,
```

Comment:
typo here

Reply:
this is an outdated  commit, fixed already [here](https://github.com/nftlabs/nftlabs-sdk-ts/blob/1cff35348c3e177b65976c6f39ccb060632e9f9b/src/common/module-type.ts#L9)